test/features/binary_build.feature

```diff
@@ -17,3 +17,17 @@ Feature: Use binary build to deploy the service
 
```

Comment:
`# process-springboot-example is used here as it is the quickest example to be built`
that comment could be removed and we could put this test as a smoke test. wdyt ?

Reply:
deleted and added @smoke